### PR TITLE
fix: handle empty name in log details avatar

### PR DESCRIPTION
**Pull request by HarshitKumar9030**
Fixed!

### CODE REVIEW

File: lib/components/log_details.dart
Comment (Sanmeet007) [inline lib/components/log_details.dart:1]: Instead of full 'Unknown' string in avatar use just '?' 

```dart
 name.isNotEmpty ? name[0] : 'Unknown',
```

**Review by Sanmeet007 — CHANGES_REQUESTED** [review]
Instead of full 'Unknown' string in avatar use just '?' 

```dart
 name.isNotEmpty ? name[0] : 'Unknown',
```

**Comment by HarshitKumar9030** [comment]
Nice app tho! I was looking for something like this for getting the logs, I actually wanna have them on a server to process further this will be a great starting point!

**Review by Sanmeet007 — APPROVED** [review]

**Comment by Sanmeet007** [comment]
Hi @HarshitKumar9030 
Thank you for spotting the bug and fixing it! Your contribution is greatly appreciated and will help improve the overall quality of the project.

Thanks again for your help!

**Pull request merged**